src/tests/ftest/util/job_manager_utils.py

```diff
@@ -297,7 +320,10 @@ def kill(self):
         if not self.job:
             return
         regex = self.job.command_regex
-        detected, running = stop_processes(self.log, self._hosts, regex)
+        if self.job.full_command_regex:
+            regex = f"'{str(self.job)}'"
```

Comment:
To my previous point, the variable name implies this should be (which is wrong)
```suggestion
        if self.job.full_command_regex:
            regex = self.job.full_command_regex
```

Reply:
Right, using `self.job.use_full_command_regex` would make more sense.